EF.msbuild

```diff
@@ -11,12 +11,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectToBuild Include="src\EntityFramework\EntityFramework.csproj" />
```

Comment:
just to clarify, reason for these projects being removed, were these projects deprecated or there are new project or components to replace them?

Reply:
The real code is in https://github.com/dotnet/ef6 (and it has been getting updates) , this was a copy to make certain methods available for writing some specific unittests.  We're moving forwards the version of the assemblies, so there are going to be differences and after weighing the alternatives it became "use the modern versions from the nuget packages"